src/test/java/TaskTest.java

```diff
@@ -0,0 +1,15 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import chatbot.task.Task;
+
+    class TaskTest {
+
+        @Test
+        void testMarkAsDone() {
```

Comment:
Maybe you can try re-naming your tests according to the `unitBeingTested_descriptionOfTestInputs_expectedOutcome` convention?
```suggestion
        void taskMarker_inputTask_taskMarkedAsDone() {
```